translations/pt-br/04-ferramentas-utilizadas.md

```diff
@@ -0,0 +1,142 @@
+# Ferramentas utilizadas
+
+Para ter sucesso na sua jornada de código aberto, é essencial familiarizar-se com as ferramentas e plataformas comumente utilizadas pela comunidade. Essas ferramentas ajudarão você a colaborar de forma eficaz, gerenciar seus projetos e contribuir para projetos de código aberto. Neste capítulo, discutiremos as seguintes ferramentas essenciais para colaboração de código aberto:
+
+- GitHub & Git
+- CodeSee
+- VS Code (O editor)
+- Discord
+
+## Pré-requisitos
+
+Antes de completar o seguinte passo a passo, complete o seguinte:
```

Comment:
```
Antes de completar o seguinte passo a passo, complete o seguinte:
```
I'd say to avoid repetition "completar" and "complete" but the source material it's no different.

Reply:
I agree with you that we should use the original git terms for the reasons you said.